### PR TITLE
Added: MultisegmentWellEquations

### PR DESCRIPTION
**Pull request by akva2**
This adds a separate class for the msw equation system similar to stw.

As in the earlier work, all member variables are still public, will be taken care of later.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: opm/simulators/wells/MultisegmentWellEquations.cpp [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+{
+#if HAVE_UMFPACK
+    if (duneDSolver_) {
+        return;
+    }
+
+    duneDSolver_ = std::make_shared<Dune::UMFPack<DiagMatWell>>(duneD_, 0);
+#else
+    OPM_THROW(std::runtime_error, "MultisegmentWell support requires UMFPACK."
+              "Reconfigure opm-simulators with SuiteSparse/UMFPACK support and recompile.");
```
Comment (bska) [inline opm/simulators/wells/MultisegmentWellEquations.cpp:166]: I take it this message is unchanged from its original form, but maybe we can use this opportunity to add a space after the full stop.  As or right now, the diagnostic message is
```
... requires UMFPACK.Reconfigure opm-simulators ...
```
and that is less than ideal in my opinion.

**Review by bska — COMMENTED** [review]
This looks good to me.  One tiny nit in a diagnostic message that few people will ever see, but otherwise I'm prepared to merge this.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the update.  I'll merge into master.

**Pull request merged**